src/components/ListingCard/index.tsx

```diff
@@ -25,20 +25,42 @@ export default function ListingCard({
       onClick={onClick ? () => onClick(caseData.id) : undefined}
     >
       {/* hard-coded for now */}
-      <H2>Case title.</H2>
-      <p>
-        <strong>Languages: </strong>
-        {caseData.languages.join(', ')}
-      </p>
-      <p>
-        <strong>Upcoming Date: </strong>
-        {parseDate(timestampStringToDate(caseData.upcoming_date))}
-      </p>
+      <CardTitle>
+        Guatemalan mother and two children seeking protection
+      </CardTitle>
+
       <TagRow>
-        {caseData.needs_interpreter && (
-          <CardTag color={COLORS.interpreterColor}>Interpreter needed</CardTag>
+        {caseData.needs_interpreter && caseData.needs_attorney && (
+          <CardTag color={COLORS.tagColor}>
+            Needs Interpreter & Attorney
+          </CardTag>
+        )}
+        {caseData.needs_interpreter && !caseData.needs_attorney && (
+          <CardTag color={COLORS.tagColor}>Needs Interpreter</CardTag>
+        )}
+        {!caseData.needs_interpreter && !caseData.needs_attorney && (
+          <CardTag color={COLORS.tagColor}>Needs Attorney</CardTag>
+        )}
```

Comment:
If a case doesn't need an interpreter or an attorney, do we want to show it? (also might want to remove the `!` before the `needs_attorney` boolean here)

Reply:
Also unsure if we'll ever encounter a case where they will find an interpreter before an attorney, but good to have just in case 👍

Reply:
Also to consider: could we make a function that helps us with the logic? It could help declutter it since it is technically just one tag with the text conditional on the boolean. (see [PR28](https://github.com/calblueprint/immigration-justice-project/pull/28/files#diff-2b850697f022cdb66b964e47c0f8eee6b3f317f0d63f8d4269c895a5d64cf306), `ButtonDropdown.tsx`)

Reply:
> If a case doesn't need an interpreter or an attorney, do we want to show it? (also might want to remove the ! before the needs_attorney boolean here)

yeah let's not show it if it doesn't need either!

Reply:
> Also unsure if we'll ever encounter a case where they will find an interpreter before an attorney, but good to have just in case 👍

definitely possible! if, say, more interpreters indicate interest and we handle their interest while waiting for attorney interest or something

Reply:
sorry, that `!` was a typo

Reply:
Also, glad you mentioned the helper function, I thought of doing that initially and ended up not doing it lol. I've included it now though!